### DISCUSSION TITLE
Protobuf 3.25.5 support ?

### DISCUSSION THREAD

**Discussion by umutsahin** [valkey-io/valkey#1417] [Q&A] [answered]
Hi there,

First of all I must say I just did a PoC with Valkey Cluster (bitnami chart) with Jedis forked client and results look amazing.
I was wondering to test valkey-glide and its features well with quarkus, but I am not able to use it due to version incompatibility since quarkus using 3.25.5 right now.

I was thinking if I would be able to compile glide with libprotoc 25.5 so that I would be able to test it.
But I am not able to build it successfully so far. Can you guide me even if I am able to compile it, will it be possible to use the client ?

Thanks!

**Comment by zuiderkwast** [comment]
Valkey itself is not using protobuf. It is only used within in Glide. You should open an issue on the Glide repo.

**Comment by avifenesh** [comment]
@umutsahin please open it again in valkey-glide for visibility.
Anyway, to your question, we are shadowing protobuf in Java client so it would work with different dependencies versions of protobuf.
From 1.3 it wont be a problem anymore. 1.3 is about to be released in 1-2 weeks.
If you still interested in compiling it please join the Valkey slack and lets discuss it.

**Reply by avifenesh** [reply]
https://join.slack.com/t/valkey-oss-developer/shared_invite/zt-2nxs51chx-EB9hu9Qdch3GMfRcztTSkQ

**Reply by umutsahin** [reply, marked as answer]
I had opened it already at the same day I created this, but forgot to close this one, sorry for the inconvenience.
https://github.com/valkey-io/valkey-glide/discussions/2804

**Reply by zuiderkwast** [reply]
Thanks, it's always good to cross-link things so it's easier to find for others.
